Assets/Scripts/Augment/ProjectileController.cs

```diff
@@ -136,6 +136,11 @@ protected virtual void Awake()
             Debug.Log("Barrel not attached to gun parent!");
             return;
         }
+        if (!gunController.player)
+        {
+            Debug.Log("Barrel w/o player reference, is this a display gun?");
```

Comment:
I think we should remove this debug so ut doesn't clutter the consoler everytime we enter the FPS scenes

Reply:
Done.

Reply:
Ye, we should probably go on a debug hunt soon (tm)